pkg/controller/direct/dataform/repository_controller.go

```diff
@@ -188,7 +189,17 @@ func (a *Adapter) Create(ctx context.Context, u *unstructured.Unstructured) erro
 func (a *Adapter) Update(ctx context.Context, u *unstructured.Unstructured) error {
 
 	updateMask := &fieldmaskpb.FieldMask{}
-	// TODO(acpana): handle updates
+
+	if a.desired.Spec.GitRemoteSettings != nil {
```

Comment:
Somewhere (maybe in an open PR) we have a helper to compute these, assuming we want the top-level fields.

Not a blocker